### DISCUSSION TITLE
On the question of multilingualism

### DISCUSSION THREAD

**Discussion by DyingStar314** [Tetrakern/fictioneer#30] [General]
Hello, I see that there is a language directory in the repository, but I don't see a language file, I would like to ask if there are any plans to achieve multilingualism in the near future? This is because it can be used better by people who are not native English speakers. I also want to help translate into my own native language so that I can use it better for myself and others who speak the same language as me
(The content is translated by translation software, please forgive me for any offense)

**Comment by Tetrakern** [comment]
Hello!

The theme is _translation-ready_, which means everything is prepared for translations to be added. However, as you have noticed, no translation is currently part of the theme. I mean, where would they come from? If you want to translate the theme, you can use a [translation plugin](https://wordpress.org/plugins/tags/translation/) to generate the required files.

But personally, I will not invest any effort into translation the theme. I couldn't even be bothers to translate it into my own language, which is not English. I have no way to quality-control the translations anyway. Maybe at some point in the future, someone will join as contributor and take care of that. But I'll think about adding a template file, at least.

**Reply by DyingStar314** [reply]
Thank you very much for your reply. I agree with you that translation is really something that requires a lot of effort, and it is incredible that you have to maintain a lot of translations by yourself. So I think we can ask for translation help on the homepage or in the document or tell people that this project can be translated and let the community do the translation task. You also mentioned about the quality of translation, I personally think that most people will not do malicious translation or vandalism, because it is basically only the speakers of the language who can translate the language. In the future, you can also consider looking for translation contributors who are willing to conduct translation audits to increase the accuracy and security of the translation

**Comment by Tetrakern** [comment]
I have added a template .pot file to the language folder.

**Reply by wpallc** [reply]
Not sure how to proceed with this, but I'd like to create translations of the Fictioneer theme. I'm doing it for my site and can offer them to the project. I got the .pot file and will have new language .po and .mo files ready soon. How do I become a contributor to this project?

**Reply by Tetrakern** [reply]
Branch the repo, add your changes, then make a pull request against the main branch of this repo. I'll review the PR and merge it before the next release. Well, I'll have an AI run over it to look for malicious and broken strings, I'm not going to evaluate your translations. GitHub will then list you as contributor automatically.

**Reply by wpallc** [reply]
Thanks, will do! Still translating...
